# Generated by Django 2.1.3 on 2018-11-29 20:48

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('awards', '0005_change_models'),
    ]

    operations = [
        migrations.CreateModel(
            name='ApiError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('ts', models.DateTimeField(auto_now_add=True)),
                ('origin', models.CharField(max_length=30)),
                ('destination', models.CharField(max_length=30)),
                ('date', models.DateField(default=datetime.date.today)),
                ('error_str', models.CharField(max_length=1000)),
            ],
        ),
    ]
